moa/src/main/java/com/yahoo/labs/samoa/instances/FilteredSparseInstance.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * 	        http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the
+ * License.  
+ */
+package com.yahoo.labs.samoa.instances;
+
+import moa.streams.ExampleStream;
+
+/**
+ * The Class FilteredSparseInstance.
+ *
```

Comment:
Please, can you add a short explanation of the motivation of this class and why it's needed?